# ENGS93: Statistical Methods in Engineering:
## <span class="mark">HOMEWORK 4 <span class="mark">SOLUTIONS</span></span>

## QUESTION 1

Load code from RX2:

In [14]:
normtemp <- read.delim("https://raw.githubusercontent.com/difuse-dartmouth/ENGS93/main/normtemp.txt", header = TRUE, sep = "\t", dec = ".")
fat <- read.delim("https://raw.githubusercontent.com/difuse-dartmouth/ENGS93/main/fat.txt", header = TRUE, sep = "\t", dec = ".")
summary(normtemp)
summary(fat)
attach(normtemp)
attach(fat)

  temperature         gender          hr       
 Min.   : 96.30   Min.   :1.0   Min.   :57.00  
 1st Qu.: 97.80   1st Qu.:1.0   1st Qu.:69.00  
 Median : 98.30   Median :1.5   Median :74.00  
 Mean   : 98.25   Mean   :1.5   Mean   :73.76  
 3rd Qu.: 98.70   3rd Qu.:2.0   3rd Qu.:79.00  
 Max.   :100.80   Max.   :2.0   Max.   :89.00  

      case          fat.Brozek       fat.Siri        density     
 Min.   :  1.00   Min.   : 0.00   Min.   : 0.00   Min.   :0.995  
 1st Qu.: 63.75   1st Qu.:12.80   1st Qu.:12.47   1st Qu.:1.041  
 Median :126.50   Median :19.00   Median :19.20   Median :1.055  
 Mean   :126.50   Mean   :18.94   Mean   :19.15   Mean   :1.056  
 3rd Qu.:189.25   3rd Qu.:24.60   3rd Qu.:25.30   3rd Qu.:1.070  
 Max.   :252.00   Max.   :45.10   Max.   :47.50   Max.   :1.109  
      age            weight      unknown_measurement      BMI       
 Min.   :22.00   Min.   :118.5   Min.   :29.50       Min.   :18.10  
 1st Qu.:35.75   1st Qu.:159.0   1st Qu.:68.25       1st Qu.:23.10  
 Median :43.00   Median :176.5   Median :70.00       Median :25.05  
 Mean   :44.88   Mean   :178.9   Mean   :70.15       Mean   :25.44  
 3rd Qu.:54.00   3rd Qu.:197.0   3rd Qu.:72.25       3rd Qu.:27.32  
 Max.   :81.00   Max.   :363.1   Max.   :77.75       Max.   :48.90  
 fat.free.weight      neck           chest           ab

The following objects are masked from normtemp (pos = 4):

    gender, hr, temperature


The following objects are masked from normtemp (pos = 6):

    gender, hr, temperature


The following objects are masked from fat (pos = 4):

    abdomen, age, ankle, biceps, BMI, case, chest, density, fat.Brozek,
    fat.free.weight, fat.Siri, forearm, hip, knee, neck, thigh,
    unknown_measurement, weight, wrist


The following objects are masked from fat (pos = 6):

    abdomen, age, ankle, biceps, BMI, case, chest, density, fat.Brozek,
    fat.free.weight, fat.Siri, forearm, hip, knee, neck, thigh,
    unknown_measurement, weight, wrist




# a)

In [15]:
mu.temp=98.2
xbar.temp=mean(temperature)
s.temp=sd(temperature)
z.temp=(xbar.temp-mu.temp)/(s.temp/sqrt(length(temperature)))
pval=pnorm(-z.temp)*2 #pval=0.4439208
#Or we can use the following command:t.test(temperature, mu =98.2, alt ="two.sided")


We get p-value =0.4453. As before 0.4453>0.4439208 because of fatter tail.
In either case, the difference is not statistically significant at alpha=0.05 level.

# b) 

In [16]:
females =temperature[gender ==1]
males =temperature[gender ==2]
#H0: mu_f=mu_m
#H1: mu_f\=mu_md
delta.gender=0
xbar.females=mean(females)
xbar.males=mean(males)
s.females=sd(females)
s.males=sd(males)
z.gender=(xbar.females-xbar.males-delta.gender)/sqrt(((s.females^2)/length(females))+((s.males^2)/length(males)))
pval=pnorm(z.gender)*2 #pval=0.02228736

#output:
pval

[1] 0.02228736

Since the p value is smaller than 0.05 we conclude that we reject the null hypothesis that the body temperatures of males and females are the same.

# c)

i)

Null hypothesis (H0): mu_BMI <= 25
Alternate hypothesis (H1): mu_BMI > 25

ii)

In [17]:
mu.BMI=25
xbar.BMI=mean(BMI)
s.BMI=sd(BMI)
z.BMI=(xbar.BMI-mu.BMI)/(s.BMI/sqrt(length(BMI)))
pval=pnorm(-z.BMI) #pval = 0.02864043
#OR
t.test(BMI, mu =25, alt ="greater")#=0.02921


	One Sample t-test

data:  BMI
t = 1.9012, df = 251, p-value = 0.02921
alternative hypothesis: true mean is greater than 25
95 percent confidence interval:
 25.0575     Inf
sample estimates:
mean of x 
  25.4369 


iii)

The two methods give slightly different p values and the first one is smaller than the second one as expected.
Either way, we find that we reject the null hypothesis that BMI <= 25.So the difference is statistically significant at 5% level (or 10% level).

# d)

In [18]:
t.test(males, females, alt="two.sided")
t.test(males, females, alt="less")
t.test(males, females, alt="greater")
t.test(females, males, alt="two.sided")



	Welch Two Sample t-test

data:  males and females
t = 2.2854, df = 127.51, p-value = 0.02394
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 0.03881298 0.53964856
sample estimates:
mean of x mean of y 
 98.39385  98.10462 



	Welch Two Sample t-test

data:  males and females
t = 2.2854, df = 127.51, p-value = 0.988
alternative hypothesis: true difference in means is less than 0
95 percent confidence interval:
      -Inf 0.4989169
sample estimates:
mean of x mean of y 
 98.39385  98.10462 



	Welch Two Sample t-test

data:  males and females
t = 2.2854, df = 127.51, p-value = 0.01197
alternative hypothesis: true difference in means is greater than 0
95 percent confidence interval:
 0.07954459        Inf
sample estimates:
mean of x mean of y 
 98.39385  98.10462 



	Welch Two Sample t-test

data:  females and males
t = -2.2854, df = 127.51, p-value = 0.02394
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -0.53964856 -0.03881298
sample estimates:
mean of x mean of y 
 98.10462  98.39385 


Analysis:

alt=”two.sided”:We get a p value of 0.02394, which is close to what we got earlier in question 6 (0.02228736). First one is smaller as expected.

alt="less": We get a p value of 0.988 which is approximately equal to 1-previous value/2=1-0.02394/2=0.98803

alt="greater": We get a p value of 0.01197 which is 1-what we got for less.

reverse order: Upon reversing the order of the variables male and female, we get the same p value but the t value is negative of what we had before.#All of these should make sense to you knowing once you think of a normal distribution pdf and the various probabilities associated with different areas under it.
